.github/workflows/commercial-bundle-bump.yml

```diff
@@ -0,0 +1,107 @@
+name: Bump @guardian/commercial-bundle
+
+on:
+    schedule:
+        # runs every 5 minutes
+        - cron: '*/5 * * * *'
```

Comment:
Is it worth changing this cron expression to run every 5 mins within some standard workday? I don't think it'd have a cost benefit since this is a public repo, but it might avoid spinning up lots of GHA containers during times there's likely not to be new commercial bundle versions?

Reply:
That's a great and easy optimisation idea thanks!

Reply:
Thanks!